src/test/java/com/epam/tc/hw2/Ex1.java

```diff
@@ -0,0 +1,120 @@
+package com.epam.tc.hw2;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+
+public class Ex1 extends Abstract {
+
+    @Test
+    //1-2.  Opening test site and assert browser title
+    public void homePage() {
+        assertThat(driver.getTitle()).isEqualTo("Home Page");
+    }
+
+    @Test
+    //3-4. Login and assert username is loggined
+    public void userLogged() {
+        WebElement login = driver.findElement(By.id("user-icon"));
+        login.click();
+        new WebDriverWait(driver, 10).until(ExpectedConditions.presenceOfElementLocated(By.id("user-icon")));
+        WebElement username = driver.findElement(By.id("name"));
+        username.sendKeys("Roman", Keys.TAB, "Jdi1234", Keys.ENTER);
+        By name = By.id("user-name");
+        assertThat(driver.findElement(name).getText()).isEqualTo("ROMAN IOVLEV");
+    }
+
+    @Test
+    //5. Assert 4 items in the header section with proper text
+    public void indexPageItems() {
+        List<String> headerNames = new ArrayList<>();
+        headerNames.add("HOME");
+        headerNames.add("CONTACT FORM");
+        headerNames.add("SERVICE");
+        headerNames.add("METALS & COLORS");
+
+        List<WebElement> header = driver.findElements(
+                By.xpath("//ul[@class=\"uui-navigation nav navbar-nav m-l8\"]/li/a"));
+        List<String> itemsHeaderNames = new ArrayList<>();
```

Comment:
You can use streams with map() here to receive list of strings from list of elements